common/apps/apps.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package apps
+
+import (
+	"github.com/coreos/rocket/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+)
+
+type App struct {
+	image string   // the image as-supplied by the user on the cli
+	args  []string // any arguments the user supplied for this app
+	asc   string   // signature file override for image verification (if fetching occurs)
+
+	imageID types.Hash // resolved image identifier
+}
```

Comment:
Since these are exposed via Get/Set, I would just make them public and not do the Javaisque getters and setters.